### ISSUE TITLE
Support for expand in the fields

### ISSUE THREAD

**Issue by mateuszlewko** [david-plugge/typed-pocketbase#17]
Hey!

Many thanks for this awesome library. I think it should be bundled with PocketBase js sdk.
Is it possible to support typing "expand" in the fields function?

```typescript
await db.collection('coll1').getFullList({
    fields: fields('id', 'name', 'created', 'expand'), // <-- this works but doesn't type.
    expand: expand({ nested: true })
});
```

**Comment by david-plugge** [comment]
Glad you like it! 
How exactly does that work? Can you give an example with the database structure, query and result?

**Comment by mateuszlewko** [comment]
It's document in API preview for each collection.
<img width="934" alt="Screenshot 2023-09-27 at 16 59 22" src="https://github.com/david-plugge/typed-pocketbase/assets/6179597/987bea9e-2494-4492-b030-ffbc0c0a62e8">

For the example query above PB will return this:
```
{
    "created": "2023-09-26 19:44:25.101Z",
    "expand": {
        "nested": {
            "collectionId": "garn2zqvrx5etf1",
            "collectionName": "services",
            "created": "2023-09-26 19:44:15.498Z",
            "creator": "nqin9tg31ug9ex7",
            "id": "75or0smdif88hzl",
            "name": "SvcA",
            "updated": "2023-09-26 19:44:15.498Z"
        }
    },
    "id": "x2gyrks4zja6zcj",
    "name": "Dep1"
}
```


For 
```await db.collection('deployments').getFullList({
		fields: fields('id', 'name', 'created', 'expand.service.name' as any),
		expand: expand({ service: true })
	});
```
it will only return service name in the expanded object.

We can  also specify `expand.nested.expand.field`, which works as expected.

**Comment by david-plugge** [comment]
This is available in `0.1.0`!
